topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.md

```diff
@@ -0,0 +1,321 @@
+>             outputs of **FastQC** {% icon tool %})
+>
+> 3. Using the {% icon galaxy-eye %} button, inspect the *Webpage* output produced by the tool
+>
+>    > ### {% icon question %} Questions
+>    >
+>    > 1. Based on the report, do you think preprocessing of the reads
+>    >    (trimming and/or filtering) will be necessary before mapping?
+>    >
+>    > > ### {% icon solution %} Solution
+>    > >
+>    > > 1. Sequence quality is quite good overall. If anything you might
+>    > >    consider trimming the 3' ends of reads (base qualities decline
+>    > >    slightly towards the 3' ends) or to filter out the small fraction
+>    > >    of reads with a mean base quality < 5.
+>    > >    We will run **Trimmomatic** {% icon tool %} on the
+>    > >    fastq datasets in the next step.
+>    > >
+>    > {: .solution}
+>    {: .question}
+>
+> 4. Use **Trimmomatic** {% icon tool %} to clean up the reads and remove the poor quality sections.
+>       - *"Single-end or paired-end reads?"*: `Paired End (two separate input files)`
+>       - {% icon param-files %} *"Input FASTQ file (R1/first of pair)"*: `ERR550641_1.fastq.gz`
+>       - {% icon param-files %} *"Input FASTQ file (R2/second of pair)"*: `ERR550641_2.fastq.gz`
+>       - *Select Trimmomatic operation to perform*
+>           - Keep the default value of **Sliding window trimming** and adjust the average quality required to 30
+>       - *"+Insert Trimmomatic Operation"*
+>           - *"Select Trimmomatic operation to perform"*: `Drop reads below a specified length (MINLEN)`
+>           - *"Minimum length of reads to be kept"*: `20`
+>
+> 5. Inspect the output produced by Trimmomatic
+>
+>    > ### {% icon question %} Questions
+>    >
+>    > 1. Why are there 4 output read files instead of 2?
+>    >
+>    > > ### {% icon solution %} Solution
+>    > >
+>    > > 1. There are 4 output files: Forwards paired and single reads and reverse paired and single reads. The single reads come about when one read in a pair of reads has failed the quality checks and so is deleted. The other half of the pair may still be good and so it is put into the single reads file for the appropriate direction.
+>    > >
+>    > {: .solution}
+>    {: .question}
+{: .hands_on}
+
+# Look for contamination with Kraken2
+
+We should also look for contamination in our reads. Sometimes, other sources of DNA accidentally or inadvertantly get mixed in with our sample. Any reads from non-sample sources will confound our snp analysis. **Kraken 2** is an effective way of looking and which species is represented in our reads and so we can easily spot possible contamination of our sample.
+
+> ### {% icon hands_on %} Hands-on: Run Kraken2
+>
+> 1. **Kraken2** {% icon tool %} with the following parameters
+>   - *"Single or paired reads"*: `Paired`
+>       - *"Forward Strand"*: `Trimmomatic on https://zenodo.org/record/3531703/files/ERR550641_1.fastq.gz (R1 paired)`
```

Comment:
I think there is a tab problem here (at least looking at the rendering on https://slugger70.github.io/training-material/topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.html)